tests/stubtest_whitelists/py3_common.txt

```diff
@@ -247,6 +247,7 @@ io.BufferedReader.truncate
 io.BufferedWriter.seek
 io.BufferedWriter.truncate
 io.BytesIO.readlines
+io.BytesIO.seek
```

Comment:
Same question here. Probably in the future it's going to be a good practice to add an explanatory comment whenever we add something to the whitelist.

Reply:
```suggestion
io.BytesIO.seek  # Parameter name for a positional-only param differs from its name in the inherited method
```